src/core/models/alternativeverticals.js

```diff
@@ -13,15 +13,15 @@ export default class AlternativeVerticals {
 
   /**
    * Create alternative verticals from server data
-   * @param {Object} response The server response
-   * @param {Object.<string, function>} formatters The field formatters to use
+   * @param {Object[]} alternativeVerticals
    */
-  static from (response, formatters) {
-    const alternativeVerticals = response.alternativeVerticals;
-    if (!alternativeVerticals || !alternativeVerticals.modules) {
+  static from (alternativeVerticals) {
+    if (!alternativeVerticals || alternativeVerticals.length < 0) {
       return new AlternativeVerticals();
     }
 
-    return new AlternativeVerticals(Section.from(alternativeVerticals.modules, {}, formatters));
+    return new AlternativeVerticals(alternativeVerticals.map(alternativeVertical => {
```

Comment:
Is it possible to replace this arrow function with just `VerticalResults.fromCore`?

Reply:
Not really, it is functionally different since `VerticalResults.fromCore` has more than one argument and we'd like the defaults applied for every arg except the first